Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -3125,7 +3125,7 @@ Finally, see if you can play with the version number of a man-in-the-middled app
 
 ### References
 
-- [#THIEL] Thiel, David. iOS Application Security: The Definitive Guide for Hackers and Developers (Kindle Locations 3394-3399). No Starch Press. Kindle Edition.
+- [#THIEL] David Thiel, iOS Application Security: The Definitive Guide for Hackers and Developers (Kindle Locations 3394-3399), No Starch Press, Kindle Edition.
```

Comment:
please make THIEL lowercase

Reply:
Done. Added as thiel2 as it is second refernce.